package.json

```diff
@@ -34,25 +34,30 @@
     }
   },
   "dependencies": {
-    "@itwin/core-bentley": "3.7.13",
-    "@itwin/core-common": "3.7.13",
-    "@itwin/core-frontend": "3.7.13",
-    "@itwin/core-geometry": "3.7.13",
-    "@itwin/core-i18n": "3.7.13",
-    "@itwin/core-markup": "3.7.13",
-    "@itwin/core-orbitgt": "3.7.13",
-    "@itwin/core-quantity": "3.7.13",
-    "@itwin/presentation-common": "3.7.13",
-    "@itwin/presentation-frontend": "3.7.13",
-    "@itwin/webgl-compatibility": "3.7.13",
+    "@itwin/core-bentley": "4.1.0",
```

Comment:
why are we pinning deps?

Reply:
Because mobile releases have to have a specific add-on version, and the only way to guarantee that is to pin the deps.